src/status_im/utils/keychain/core.cljs

```diff
@@ -155,19 +155,36 @@
              (callback nil))))
         (callback nil))))))
 
+(defn save-user-password!
+  [key-uid password]
+  (save-credentials
+   key-uid
+   key-uid
```

Comment:
why does this function take the same param twice? 😵‍💫

Reply:
ah I see it's a logging message type thing 👍

Reply:
I have no idea to be honest, I didn't want to change it :) , just to make sure

Reply:
I had the same question when I saw this code researching keychain and biometrucs old code, we can revisit it later and refactor if needed